From c680934c282c791f151f4efe173c53604bac7c93 Mon Sep 17 00:00:00 2001
From: Jason King <jason.king@joyent.com>
Date: Sat, 14 Apr 2018 12:31:45 -0500
Subject: [PATCH] OS-6891 usr/src/lib/fm is missing a dependency on libpcidb
 Reviewed by: Patrick Mooney <pmooney@pfmooney.com> Approved by: Mike Gerdts
 <mike.gerdts@joyent.com>

---
 usr/src/lib/Makefile | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/usr/src/lib/Makefile b/usr/src/lib/Makefile
index 445238f973..7d337e7175 100644
--- a/usr/src/lib/Makefile
+++ b/usr/src/lib/Makefile
@@ -588,7 +588,8 @@ auditd_plugins: libbsm libsecdb libgss libmtmalloc
 brand:		libzonecfg libmapmalloc libipadm libcmdutils libproc librpcsvc
 cfgadm_plugins:	libdevice libdevinfo libhotplug librcm hbaapi libkstat libscf
 fm:		libexacct libipmi libzfs scsi libdevinfo libdevid libcfgadm \
-		libcontract libsysevent ../cmd/sgs/libelf libdladm libsff
+		libcontract libsysevent ../cmd/sgs/libelf libdladm libsff \
+		libpcidb
 $(SPARC_BLD)fm:	libpri
 gss_mechs/mech_dh: libgss
 gss_mechs/mech_dummy: libgss
-- 
2.21.0

